### PR TITLE
add note about the style guide to the contributors guide

### PR DESCRIPTION
**Pull request by kdoh**

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ To get a sense for the end goals, make sure to read the [Zen of Metabase](../zen
 
 ## Our product process:
 
-The core team runs a pretty well defined product process. It is actively being tweaked, but the below is a pretty faithful description of it at the time of writing. You should have a clear idea of how we work before jumping in with a PR.
```
Comment (salsakran) [inline docs/contributing.md]: really not a fan of trailing spaces are we now?

File: docs/contributing.md [hunk 2]
```diff
@@ -29,22 +29,25 @@ We typically will collect a group of issues or suggestions into a new topline fe
...
 
 ### E) Verification and merging
 
 All PRs that involve more than an insignificant change should be reviewed. See our [Code Review Process](code-reviews.md).
- 
-If all goes well, the feature gets coded up, verified and then the pull request gets merged! High-fives all around. 
 
-If there are tests missing, code style concerns or specific architectural issues in the pull request, they should be fixed before merging. We have a very high bar on both code and product quality and it's important that this be maintained going forward, so please be patient with us here. 
+If all goes well, the feature gets coded up, verified and then the pull request gets merged! High-fives all around.
+
```
Comment (salsakran) [inline docs/contributing.md]: but you like newlines? ;)

**Review by salsakran — APPROVED** [review]
Looks good!

File: docs/contributing.md (hunk 2 above)
Comment (kdoh) [reply docs/contributing.md]: https://github.com/metabase/metabase/blob/a946e53c1d7fb51f14535e4766cfc1467a824c43/.editorconfig#L9

**Pull request merged**